substrate-node/runtime/src/lib.rs

```diff
@@ -487,31 +486,6 @@ pub type CurrencyTransactor = CurrencyAdapter<
 /// MultiLocation, And convert Asset transfer amount to Balance
 pub struct SimpleForeignAssetConverter(PhantomData<()>);
 
-impl Convert<MultiLocation, AssetId> for SimpleForeignAssetConverter {
-	fn convert_ref(id: impl Borrow<MultiLocation>) -> result::Result<AssetId, ()> {
-		if &UsdcLocation::get() == id.borrow() {
-			Ok(UsdcAssetId::get())
-		} else if &ERC20TSTLocation::get() == id.borrow() {
-			Ok(ERC20TSTAssetId::get())
-		} else if &ERC20TSTD20Location::get() == id.borrow() {
-			Ok(ERC20TSTD20AssetId::get())
-		} else {
-			Err(())
-		}
-	}
-	fn reverse_ref(what: impl Borrow<AssetId>) -> result::Result<MultiLocation, ()> {
-		if *what.borrow() == UsdcAssetId::get() {
-			Ok(UsdcLocation::get())
-		} else if *what.borrow() == ERC20TSTAssetId::get() {
-			Ok(ERC20TSTLocation::get())
-		} else if *what.borrow() == ERC20TSTD20AssetId::get() {
-			Ok(ERC20TSTD20Location::get())
-		} else {
-			Err(())
-		}
-	}
-}
-
 impl MatchesFungibles<AssetId, Balance> for SimpleForeignAssetConverter {
```

Comment:
I remember this piece of logic is manually linking the AssetID with asset MultiLocation, I don't see a replacement in this PR, how do we handle the connection between AssetID and asset MultiLocation if we remove it? 

I saw it's been removed in bridge mock.rs as well

Reply:
It looks like XCM has changed some implementation code that we don't need to convert AssetId to MultiLocation any more, since trait `Convert` doesn't exist in XCM any more, so I just delete it